phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateSchemaIT.java

```diff
@@ -95,9 +96,33 @@ public void testCreateSchema() throws Exception {
             conn.createStatement().execute("CREATE SCHEMA \""
                     + SchemaUtil.HBASE_NAMESPACE.toUpperCase() + "\"");
 
-            assertNotNull(admin.getNamespaceDescriptor(SchemaUtil.SCHEMA_FOR_DEFAULT_NAMESPACE.toUpperCase()));
-            assertNotNull(admin.getNamespaceDescriptor(SchemaUtil.HBASE_NAMESPACE.toUpperCase()));
+            assertTrue(isHbaseNamespaceAvailable(admin, SchemaUtil.SCHEMA_FOR_DEFAULT_NAMESPACE.toUpperCase()));
+            assertTrue(isHbaseNamespaceAvailable(admin, SchemaUtil.HBASE_NAMESPACE.toUpperCase()));
 
         }
     }
+
+    /**
+     * Returns true if HBase namespace exists, else returns false
+     * @param hbaseAdmin HbaseAdmin Object
+     * @param schemaName Phoenix schema name for which we check existence of the HBase namespace
+     * @return true if the HBase namespace exists, else returns false
+     * @throws SQLException If there is an exception checking the HBase namespace
+     */
+    boolean isHbaseNamespaceAvailable(HBaseAdmin hbaseAdmin, String schemaName) throws IOException{
```

Comment:
Rather than duplicating the code between several tests, better to have a common place the function lives. For example, can just call the version of it in CQS, or move the version you added in CQS to a util class somewhere.